poetry/core/version/markers.py

```diff
@@ -157,6 +157,10 @@ def _format_marker(marker, first=True):
 
 
 class BaseMarker(object):
+    @property
+    def inverse(self):  # type: () -> BaseMarker
```

Comment:
@sdispater since this is returning a new object by performing an inversion should we a method instead of a property? Non-blocker, but might be a good idea.

Reply:
I changed it to a `invert()` method.